hack/make-helm-chart.sh

```diff
@@ -25,9 +25,9 @@ TAG="${TAG:?TAG env variable must be specified}"
 HELM_CHART_REPO="us-docker.pkg.dev/online-boutique-ci/charts"
 
 cd helm-chart
-sed -i "s/^appVersion:.*/appVersion: \"${TAG}\"/" Chart.yaml
-sed -i "s/^version:.*/version: ${TAG:1}/" Chart.yaml
+gsed -i "s/^appVersion:.*/appVersion: \"${TAG}\"/" Chart.yaml
+gsed -i "s/^version:.*/version: ${TAG:1}/" Chart.yaml
```

Comment:
We `hack` script [uses `gsed`](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/hack#hack) instead of `sed`.
Using `gsed` allows us to run the script `sed` commands on both Linux _and_ Mac.